src/main/java/core/basesyntax/MyHashMap.java

```diff
@@ -1,19 +1,102 @@
 package core.basesyntax;
 
+import java.util.Objects;
+
 public class MyHashMap<K, V> implements MyMap<K, V> {
+    private static final int DEFAULT_CAPACITY = 16;
+    private static final float LOAD_FACTOR = 0.75f;
+    private static final int MULTIPLAYER = 2;
+
+    private Node<K, V>[] buckets;
+    private int threshold;
+    private int size;
+
+    public MyHashMap() {
+        this.buckets = new Node[DEFAULT_CAPACITY];
+        this.threshold = (int) (DEFAULT_CAPACITY * LOAD_FACTOR);
+    }
 
     @Override
     public void put(K key, V value) {
-
+        Node<K, V> newNode = new Node<>(getHash(key), key, value, null);
+        int index = Math.abs(newNode.hash % buckets.length);
```

Comment:
I suggest you to make separate method getIndex and use it in the different methods

Reply:
done